APPLICATION_PROGRESS.md

```diff
@@ -422,16 +422,17 @@ the sdk
 <details>
 <summary>Over the counter (OTC) Swap Predicate</summary>
 
-<h3>Contracts</h3>
+<h3>Contracts✅</h3>
```

Comment:
Given the comment below I would mention the todo in config time constants but probably leave this as a check mark since that seems to be more code quality than functionality